src/database/knex/migrations/20200902234100_current_semester.ts

```diff
@@ -0,0 +1,32 @@
+import * as Knex from 'knex'
+
+export async function up (knex: Knex): Promise<void> {
+  return knex.schema.createTableIfNotExists('currentSemester', function (table) {
+    table.increments('id').primary()// PK of table Current Semester
+
+    table.integer('idCalender').notNullable()
+    table.integer('idDicipline').notNullable()
+
+    table.timestamp('createdAt', { precision: 6 }).defaultTo(knex.fn.now(6))
+    table.timestamp('lastUpdate', { precision: 6 }).defaultTo(knex.fn.now(6))
+
+    table.timestamp('evalP1Start', { precision: 6 }).defaultTo(knex.fn.now(6))
```

Comment:
Tira os default, não faz sentido as datas de avaliação serem iniciadas com a data de hoje.
Normalmente deixamos elas agendadas com meses de diferença.
E coloca um Not Nullable pra obrigar o cara a preencher essa porra

Reply:
tira de todos os de baixo também